packages/ui/src/molecules/QuantitySelector/stories/QuantitySelector.mdx

```diff
@@ -0,0 +1,32 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs'
+
+import QuantitySelector from '../QuantitySelector'
+
+# QuantitySelector
+
+## Styleless
+
+<Canvas>
+  <Story id="molecules-quantityselector--styleless" />
+</Canvas>
+
+## Default Style
+
+<Canvas>
+  <Story id="molecules-quantityselector--default-style" />
+</Canvas>
```

Comment:
Is this working? Looking at the `QuantitySelector` story file and I didn't find the `Styleless` and `Default Style` stories. Perhaps this distinction isn't necessary because Storybook already has the option to see the component with and without the styles:

<img width="372" alt="Xnip2021-12-16_18-35-19" src="https://user-images.githubusercontent.com/15722605/146452458-a4822339-dc15-4338-ad84-caffc684c0f8.png">

Reply:
Whoops. Sorry. Leftover code from the previous PR.